rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,222 @@
+
+##### Shared Example
+
+Save the `docker-compose.yml` file below to a directory. Notice that no modifiers are added to the volume; stack scope is default behavior.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data:/data
+volumes:
+  redis-data:
+    driver: local
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that a `redis-data` volume exists. The volume name will be in the form `<stack-name>_<volume-name>_<hash>`.
+
+> **Note:** No auth restrictions are placed on stack-scoped volumes. These volumes could theoretically be mounted by other stacks within the environment if the hash was known.
+
+##### Per-Container Example
+
+Add the modifier `per_container: true` to a volume definition and each container in a service will get its own unique volume. This makes sense for distributed stateful applications such as [Apache Cassandra](http://cassandra.apache.org/) where the application itself was designed with distributed storage in mind. More generally speaking, services with containers that are not [fungible](https://en.wikipedia.org/wiki/Fungibility) would want to persist container state to separate volumes.
+
+Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  cassandra:
+    image: llparse/cassandra:v3.7
+    labels:
+      io.rancher.scheduler.global: true
+    environment:
+      CASSANDRA_CLUSTER_NAME: 'rancher'
+      HEAP_NEWSIZE: '256M'
+      MAX_HEAP_SIZE: '256M'
+    volumes:
+    - cassandra:/var/lib/cassandra
+volumes:
+  cassandra:
+    driver: local
+    per_container: true
+```
+
+Create the stack. This creates a [global service]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/scheduling/#global-service), or a service with a container on every host.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that a `cassandra` volume exists for every host. The volume name will be in the form `<stack-name>_<volume-name>_<service-index>_<uuid>`.
+
+### V1 Compose
+
+Up until this point, we've discussed volumes in context of [Docker V2 Compose](https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes-volume_driver).
+
+[Docker V1 Compose](https://docs.docker.com/compose/compose-file/compose-file-v1/#volumes-volume_driver) may also be used. Rancher passes V1 volumes directly to Docker; as such, there are no scheduling guarantees and volumes are not automatically deleted. Containers may collide with historical volumes. [Scheduling affinities](https://docs.rancher.com/rancher/v1.6/en/cattle/scheduling/#fields-1) may be used to bridge the gap, but this incurs operational overhead (labeling hosts).
```

Comment:
Can you drop the "but this incurs..." bit? If that makes the sentence short and awkward, you could rewrite as

To bridge the gap and ensure containers are schedule to the host where a V1 volume resides, scheduling affinities may be used.